ReactAndroid/src/main/java/com/facebook/hermes/reactexecutor/Android.mk

```diff
@@ -40,7 +40,7 @@ LOCAL_C_INCLUDES := $(LOCAL_PATH) $(REACT_NATIVE)/ReactCommon/jsi $(call find-no
 
 LOCAL_CPP_FEATURES := exceptions
 
-LOCAL_STATIC_LIBRARIES := libjsireact libhermes-executor-common-debug libhermes-inspector
+LOCAL_STATIC_LIBRARIES := libjsireact libhermes-executor-common-debug
```

Comment:
I believe removing it here is incorrect, as hermes-inspector is only used by debug flavour of executor

Reply:
From my understanding `libhermes-inspector` is linked into `libhermes-executor-common-debug` already so it doesn't need to be included here. I checked the code quickly and it also doesn't seem to be using classes from `libhermes-inspector` directly (no `<hermes/inspector/*>` in `ReactAndroid/src/main/java/com/facebook/hermes/reactexecutor`).

Reply:
Ah, I see, make sense :)